Makefile

```
@@ -144,6 +147,26 @@ config:
 
 status:
 	docker compose ps
+	
+	
+./kitodo/data/metadata/testdata-kitodo:
+	mkdir -p $@/images
+	for page in {00000009..00000014}; do \
+	  wget -P $@/images https://digital.slub-dresden.de/data/kitodo/LankDres_1760234508/LankDres_1760234508_tif/jpegs/$$page.tif.original.jpg; \
+	done
+
+test: test-kitodo
+
+test-kitodo: ./kitodo/data/metadata/testdata-kitodo
+# wait until Kitodo.Production directory structure is initialized
+	docker exec -t `docker container ls -qf name=kitodo-app` bash -c "/wait-for-it.sh -t 0 kitodo-app:$$APP_PORT"
+# run asynchronous ocr processing, which should return within 5 seconds with exit status 1
+	timeout --preserve-status 5 docker exec -t `docker container ls -qf name=kitodo-app` bash -c '/usr/local/kitodo/scripts/script_ocr_process_dir.sh "testdata-kitodo" 1'; test $$? -eq 1
```

Does this really work? (The POSIX sh's test builtin AFAIK does not know `-eq`. But perhaps this is a dashism?)

```suggestion
	timeout --preserve-status 5 docker exec -t `docker container ls -qf name=kitodo-app` bash -c '/usr/local/kitodo/scripts/script_ocr_process_dir.sh "testdata-kitodo" 1'; test $$? = 1
```

---

Yes it worked. It tested this with exit status 124 of timeout in my local environment.

---

Ok, then the CI uses dash by default. Better with the suggestion, that should work on both dash/ksh and bash/sh.